.golangci.yml

```diff
@@ -4,13 +4,90 @@ run:
 #   timeout: 5m
 
 linters:
+  # Disable all linters.
+  # Default: false
   disable-all: true
+  # Enable specific linter
+  # https://golangci-lint.run/usage/linters/#enabled-by-default-linters
   enable:
+    - asciicheck
+    - bidichk
+    - bodyclose
+    - contextcheck
+    - cyclop
```

Comment:
Ah this should probably be disabled for the same reason as gocyclo

Reply:
done :)